packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -138,6 +138,42 @@ export const configSchema = {
     _default: '',
     _description: 'Custom label for patient chart button',
   },
+  queueTableColumns: {
+    _type: Type.Array,
+    _description: 'Columns to appear in the queues table.',
+    _elements: {
+      id: {
```

Comment:
Do we need this property? In all of the examples, it ends up being the `headerI18nKey`...

Reply:
Yeah, we can forgo this. This means requiring the columns headers to be unique, which should be fine.